### ISSUE TITLE
Generated plugins do not have a valid reference to the DarkRift libraries

### ISSUE THREAD

**Issue by JamJar00** [DarkRiftNetworking/darkrift-cli#2]
When new plugins are generated the references to the required DarkRift libraries are invalid and cause the build to fail. This may be fixed best after #1.

**Comment by JamJar00** [comment]
Superseded by #4
